.github/workflows/ci.yml

```diff
@@ -37,18 +37,14 @@ jobs:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        php-version:
-          - "8.2"
-
     steps:
+      - uses: extractions/setup-just@v2
       - uses: actions/checkout@v3
 
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: ${{ matrix.php-version }}
+          php-version: "8.2"
```

Comment:
Since we're not running `phpstan` across multiple PHP versions, I just hardcoded the value we're using